packages/nocodb/src/models/Filter.ts

```
@@ -960,4 +984,33 @@ export default class Filter implements FilterType {
+        {
+          condition: { fk_link_col_id: linkColumnId },
+        },
+      );
+      await NocoCache.setList(
+        CacheScope.FILTER_EXP,
+        [linkColumnId],
+        filterObjs,
+      );
+    }
+
+    return filterObjs?.map((f) => this.castType(f)) || [];
```

Biome (1.9.4)

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)